stl/inc/flat_set

```diff
@@ -70,17 +65,17 @@ public:
     static_assert(random_access_iterator<iterator>, "The C++ Standard forbids containers without random "
                                                     "access iterators from being adapted. See [flatset.overview].");
 
-    _Base_flat_set() : _My_pair(_Zero_then_variadic_args_t{}) {}
+    _Base_flat_set() : _Mycont(), _Mycomp() {}
 
+    // TRANSITION, "_My_comp" may need to be copied, even in move construction / assignment.

```

Comment:
We use TRANSITION for things that need to ship in `main`, but that we hope to improve in the future. For things that must be addressed before this feature is officially merged, it seems that we should use FIXME. The distinction is that we never allow "FIXME" to reach `main`.

Reply:
(Mentioning this for the future, but I didn't push changes here.)